oneview_redfish_toolkit/blueprints/computer_system.py

```diff
@@ -185,3 +185,38 @@ def change_power_state(uuid):
         # In case of error log exception and abort
         logging.exception('Unexpected error: {}'.format(e))
         abort(status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+
+@computer_system.route(
+    "/redfish/v1/Systems/<uuid>", methods=["DELETE"])
+def remove_subscription(uuid):
+    """Removes a specific System
+
+        Args:
+            uuid: The System ID.
+
+        Exceptions:
+            HPOneViewException: When some OneView resource was not found.
+            return abort(404)
+
+            Exception: Unexpected error.
+            return abort(500)
+    """
+    try:
+        # Deletes server profile for given UUID
+        sucess = g.oneview_client.server_profiles.delete(uuid)
+
+        if not sucess:
```

Comment:
According to the python SDK, the DELETE request may return a `True` boolean value when the profile has been successfully [removed](https://github.com/HewlettPackard/python-hpOneView/blob/a44fe186f64f4035e6cce939c67e423ecd28f777/hpOneView/resources/resource.py#L269), or a task resource as the response. If we receive a `True` as response, our code will work correctly. However, when the response is a task resource, we need to check its result before sending an OK.

Reply:
Nice input. Now it's handling task as well.